qulice-checkstyle/src/main/resources/com/qulice/checkstyle/checks.xml

```diff
@@ -344,7 +344,7 @@
         </module>
         <module name="LineLength">
             <property name="max" value="80"/>
-            <property name="ignorePattern" value="^import .*$"/>
+            <property name="ignorePattern" value="^import .*$|https?:\/\/.+"/>
```

Comment:
Yeah I think the regex with `^` in front it is also way better for performance, since the regex can skip out at the very first line. And speeding up the build is super important too.